adapters/ethereum/index.js

```diff
@@ -136,6 +111,34 @@ function Adapter(opts, cfg, ethProvider) {
 			return token
 		})
 	}
+
+	this.getDepositFor = async function(channel, depositor) {
+		const ethChannel = toEthereumChannel(channel)
+		const erc20 = new Contract(channel.tokenAddr, erc20ABI, provider)
+		const outpaceBalance = await outpace.deposits(ethChannel.hashHex(), depositor)
+		const create2Addr = this.getCreate2Address(
+			cfg.ETHEREUM_CORE_ADDR,
+			cfg.SWEEPER_ADDRESS,
+			ethChannel,
+			depositor
+		)
+
+		const create2Balance = await erc20.balanceOf(create2Addr)
+		const tokenConfig = cfg.TOKEN_ADDRESS_WHITELIST[channel.tokenAddr.toLowerCase()]
+		assert.ok(tokenConfig, 'token: invalid token addr')
+
+		if (create2Balance.gt(BigNumber.from(tokenConfig.MINIMUM_DEPOSIT))) {
+			return {
+				total: new BN(create2Balance.add(outpaceBalance).toString()),
+				create2Balance: new BN(create2Balance.toString())
+			}
+		}
+
+		return {
+			total: new BN(outpaceBalance.toString()),
+			create2Balance: new BN(0)
```

Comment:
Please follow the same glossary as in the Rust implementation. We've already agreed upon the naming of this field to be `stillOnCreate2`.

Reply:
it's been changed